go-controller/pkg/util/smartnic_annotations.go

```diff
@@ -0,0 +1,128 @@
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
```

Comment:
nit: separate this with a new line, its a project package import

Reply:
will do.